.github/workflows/reusable_tutorials.yml

```diff
@@ -1,6 +1,20 @@
 name: Reusable Tutorials Workflow
 
 on:
+  workflow_dispatch:
+    inputs:
+      smoke_test:
+        required: false
+        type: boolean
+        default: true
```

Comment:
Should we remove the defaults since this will be user triggered anyways? It's likely that we'll want to use non-default values with this most of the time.

Reply:
sure I'll just make those required then